src/core/atomicdex/models/qt.orderbook.model.cpp

```diff
@@ -571,7 +568,7 @@ namespace atomic_dex
                     {
                         if (this->m_current_orderbook_kind == kind::best_orders)
                         {
-                            // SPDLOG_INFO("Removing order with UUID: {}", id);
+                            SPDLOG_INFO("Removing order with UUID: {}", id);
```

Comment:
this one is left... should we remove that log too?
it generates output like
```
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: ae3bc6fe-f8db-4787-9aea-422897cf0fde
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: b9dd38b6-443b-4f9a-9add-8f25f461971b
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: f2c242e3-db95-4342-bae4-4602c8dfe313
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 665e858d-e8b8-4105-8e74-81092360ecd0
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 14da4f59-5e87-4b71-8da9-4c5fcf00da4a
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 18697c98-adad-48c2-af9c-f2c6a78feb29
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 1f2a4aef-d1ea-4211-af77-9c0cb76ee49f
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: f8e882b5-3e82-4c42-88a2-eb448b8856c7
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: d450dab9-130b-4de2-9aad-6f26a8858b14
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 73c7df40-3aac-4d7f-9a8a-9a6f82f33d26
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 7ea70587-1b3a-451a-aa46-abfcdf65328b
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: c1ca590f-1945-45ca-b8be-01a97e039f3b
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 69e72893-c213-4b71-b37b-e7e68091be82
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 06db0b75-4b99-43e0-bc76-f3a157091c6d
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 6a51c15f-ac93-406f-aa94-55ea2f72034b
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: dd96c864-ad45-404c-9f6b-74a75a9d6a4e
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: b52b7fe5-c1bd-4791-8280-a6bd5e9008b2
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: d990b418-cc11-4c99-9740-e52cfbbbae50
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: c349498e-d8a8-43f2-92d6-4532304159b8
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 13eb9939-1ac0-4a7b-8be3-c3ade77a0c62
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 108ad2a4-e4ba-4894-b100-7f8a9ea3e848
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 765f641d-1d74-4d71-9750-08d9249655f8
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 030af626-f71f-4346-8507-41e9734a314c
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: fc73b215-5d5c-43f8-bce5-085e080a3524
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: 9049a686-092b-4edc-bcfd-c8ef26db6d20
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: e1a9db6e-f701-4d8f-9adf-9b69fe201923
[12:54:59] [info] [qt.orderbook.model.cpp:571] [2046864]: Removing order with UUID: dc636d5f-1f79-4d61-946b-5fae644ff012
```

Reply:
nice catch